opentreemap/treemap/templates/treemap/partials/collectionudf.html

```diff
@@ -44,6 +44,9 @@
     {% endfor %}
     <tr class="placeholder" style="{{ values|yesno:"display: none;," }}"><td>{% trans "No entries yet!" %}</td>
   </table>
+  <p class="text-muted">
```

Comment:
Does this `<p>` tag need to go inside an `{% if udf.model_config.warning_message %}` block?